[2023-12-01 00:36:21,969] WARNING predict() L126  Erro ao adicionar paciente 'Maria', N√£o foi poss√≠vel salvar novo item :/ - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/api/app.py L126 
[2023-12-01 00:36:21,970] INFO _log() L96   127.0.0.1 - - [01/Dec/2023 00:36:21] "[31m[1mPOST /paciente HTTP/1.1[0m" 400 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2023-12-01 07:04:38,321] ERROR log_exception() L825  Exception on /paciente [GET] - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/flask/app.py L825 
Traceback (most recent call last):
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1969, in _exec_single_context
    self.dialect.do_execute(
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 922, in do_execute
    cursor.execute(statement, parameters)
sqlite3.OperationalError: no such column: pacientes.anaemia

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/flask_openapi3/scaffold.py", line 117, in view_func
    response = func(**func_kwargs)
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/api/app.py", line 149, in get_paciente
    paciente = session.query(Paciente).filter(Paciente.name == paciente_nome).first()
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/sqlalchemy/orm/query.py", line 2748, in first
    return self.limit(1)._iter().first()  # type: ignore
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/sqlalchemy/orm/query.py", line 2847, in _iter
    result: Union[ScalarResult[_T], Result[_T]] = self.session.execute(
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 2308, in execute
    return self._execute_internal(
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 2190, in _execute_internal
    result: Result[Any] = compile_state_cls.orm_execute_statement(
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/sqlalchemy/orm/context.py", line 293, in orm_execute_statement
    result = conn.execute(
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1416, in execute
    return meth(
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/sqlalchemy/sql/elements.py", line 516, in _execute_on_connection
    return connection._execute_clauseelement(
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1639, in _execute_clauseelement
    ret = self._execute_context(
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1848, in _execute_context
    return self._exec_single_context(
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1988, in _exec_single_context
    self._handle_dbapi_exception(
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 2343, in _handle_dbapi_exception
    raise sqlalchemy_exception.with_traceback(exc_info[2]) from e
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1969, in _exec_single_context
    self.dialect.do_execute(
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 922, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such column: pacientes.anaemia
[SQL: SELECT pacientes.id AS pacientes_id, pacientes.name AS pacientes_name, pacientes.age AS pacientes_age, pacientes.anaemia AS pacientes_anaemia, pacientes.creatinine_phosphokinase AS pacientes_creatinine_phosphokinase, pacientes.diabetes AS pacientes_diabetes, pacientes.ejection_fraction AS pacientes_ejection_fraction, pacientes.high_blood_pressure AS pacientes_high_blood_pressure, pacientes.platelets AS pacientes_platelets, pacientes.sex AS pacientes_sex, pacientes.serum_creatinine AS pacientes_serum_creatinine, pacientes.serum_sodium AS pacientes_serum_sodium, pacientes.smoking AS pacientes_smoking, pacientes.time AS pacientes_time, pacientes.data_insercao AS pacientes_data_insercao 
FROM pacientes 
WHERE pacientes.name = ?
 LIMIT ? OFFSET ?]
[parameters: ('Maria', 1, 0)]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
[2023-12-01 07:04:38,328] INFO _log() L96   127.0.0.1 - - [01/Dec/2023 07:04:38] "[35m[1mGET /paciente?name=Maria HTTP/1.1[0m" 500 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2023-12-01 07:06:05,740] INFO _log() L96   127.0.0.1 - - [01/Dec/2023 07:06:05] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2023-12-01 07:06:05,770] INFO _log() L96   127.0.0.1 - - [01/Dec/2023 07:06:05] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2023-12-01 07:06:05,776] INFO _log() L96   127.0.0.1 - - [01/Dec/2023 07:06:05] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2023-12-01 07:06:05,781] INFO _log() L96   127.0.0.1 - - [01/Dec/2023 07:06:05] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/03 - Qualidade de Software - Seguran√ßa - Sistemas Inteligentes/MVP3_QSSI/venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
